js_basics/task5.js

```diff
@@ -0,0 +1,18 @@
+// circle
+const radius = 5;
+const circleArea = Math.pow(radius * Math.PI, 2 );
```

Comment:
This expression is wrong
`Math.pow(radius * Math.PI, 2 )` 

You have to use this formula 
`π * radius^2`

And you use this
`(π * radius)^2`